temoa/extensions/stochastics/stochastic_sequencer.py

```
@@ -0,0 +1,84 @@
+            raise ValueError("Stochastic mode requires a 'stochastic_config' in the TOML.")
+
+        sc_path = self.config.stochastic_config
+        if not sc_path.exists():
+            raise ValueError(f"Stochastic config file not found: {sc_path}")
+        if not sc_path.is_file():
+            raise ValueError(f"Stochastic config path is not a file: {sc_path}")
+
+        try:
+            self.stoch_config = StochasticConfig.from_toml(sc_path)
+        except Exception as e:
+            logger.exception("Failed to load stochastic config from %s: %s", sc_path, e)
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)